opengrok-indexer/src/test/java/org/opengrok/indexer/util/CtagsUtilTest.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * CDDL HEADER START
+ *
+ * The contents of this file are subject to the terms of the
+ * Common Development and Distribution License (the "License").
+ * You may not use this file except in compliance with the License.
+ *
+ * See LICENSE.txt included in this distribution for the specific
+ * language governing permissions and limitations under the License.
+ *
+ * When distributing Covered Code, include this CDDL HEADER in each
+ * file and include the License file at LICENSE.txt.
+ * If applicable, add the following below this CDDL HEADER, with the
+ * fields enclosed by brackets "[]" replaced with your own identifying
+ * information: Portions Copyright [yyyy] [name of copyright owner]
+ *
+ * CDDL HEADER END
+ */
+
+/*
+ * Copyright (c) 2019, Chris Fraire <cfraire@me.com>.
+ */
+
+package org.opengrok.indexer.util;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+import org.opengrok.indexer.condition.ConditionalRun;
+import org.opengrok.indexer.condition.CtagsInstalled;
+import org.opengrok.indexer.configuration.RuntimeEnvironment;
+
+import java.util.List;
+
+/**
+ * Represents a container for tests of {@link CtagsUtil}.
+ */
+@ConditionalRun(CtagsInstalled.class)
```

Comment:
`CtagsInstalled` is basically `RuntimeEnvironment.getInstance().validateUniversalCtags()` which relies on `CtagsUtil` functionality which is the class under test. This sounds little bit fragile to me.

Reply:
Haha I hadn't realized the convolution. Probably `CtagsInstalled` has outlived its usefulness, right?